### DISCUSSION TITLE
Online exam

### DISCUSSION THREAD

**Discussion by vik2595** [TareqMonwer/Django-School-Management#113] [Ideas]
I was actively looking for developing suck kind of open source school /university management system. 

I would like to contribute in this. 

My question:idea is 

Does this platform support online examination?

**Comment by TareqMonwer** [comment]
@vik2595 
Thanks for your interest. 
Currently, it doesn't have an online examination system. And of course, you can contribute to this project.
I'm expecting contributions TBH :|
